Create vpn-modes-exclusions.md (#249)
AdguardTeam/KnowledgeBaseVPN: docs/general/vpn-modes-exclusions.md

---
title: VPN modes and Exclusion lists
sidebar_position: 11
---

## VPN modes and Exclusion lists

This section explains how to use website and app exclusions in different VPN modes.

## VPN modes

AdGuard VPN has two modes:

1. By default, AdGuard VPN is active for all websites and apps, except for those added to the website and app exclusion lists.

2. AdGuard VPN is active *selectively*: only for the websites and apps from the exclusion lists.

:::note

You can set different VPN modes for apps and websites. For example, you can set AdGuard VPN to be active for all apps except for those added to the app exclusion list, and active only for the websites from the website exclusion list.

:::

## Exclusion lists

An exclusion list is a list of apps or websites that AdGuard VPN excludes or includes in its tunnel based on the current VPN mode.

:::note

Each VPN mode has two exclusion lists: one for apps and one for websites.
In total, there are four exclusion lists:

* App exclusion list [VPN active everywhere]
* App exclusion list [VPN active selectively]
* Website exclusion list [VPN active everywhere]
* Website exclusion list [VPN active selectively]

:::

> ## Combining VPN modes and Exclusion lists

Reviewer: а почему тут Exclusion с большой буквы, а в остальных местах с маленькой?

> The four VPN activity setups provide different levels of VPN coverage.

Reviewer: тяжеловато

> ### AdGuard VPN is active everywhere except for apps and websites from the app and website exclusion lists

Reviewer: заголовок выйдет на три строки минимум. давай сократим?

VPN mode:

* Active for all apps except app exclusions
* Active for all websites except website exclusions

Applied exclusion lists:

* App exclusion list [VPN active everywhere]
* Website exclusion list [VPN active everywhere]

This is the default VPN mode with maximum VPN protection. If you want to exclude a website or an app from AdGuard VPN, add them to the exclusion list.

**Empty exclusion lists:** Your app exclusion list and website exclusion list can be empty. If nothing is specified in the exclusion lists, AdGuard VPN will be active for all apps and websites.

**Why use this mode:** You want a blanket VPN protection for everything on your device. If a specific app or website needs to work outside the VPN, add it to Exclusions.

:::caution

If you add a browser app to the app exclusion list, AdGuard VPN won't be active for all websites opened in that browser.

:::

> ### AdGuard VPN is active only for the apps from the the app exclusion list, and for all websites except those from the website exclusion list

Reviewer: то же самое по остальным заголовкам

* Website exclusion list [VPN active everywhere]

To switch to this mode, leave AdGuard VPN active for all websites and set it to be on only for the apps from the app exclusion list. This mode provides less VPN protection coverage because you need to add apps to the app exclusion list to ensure they are protected.

This mode requires your attention to setting up your app exclusion list.

**Empty exclusion lists:** If the **app exclusion list** is **empty**, AdGuard VPN will be **inactive for both websites and apps**.

**Why use this mode:** You need a VPN for most websites, but only for a few apps.

:::caution

If you don’t add a browser app to the app exclusion list, AdGuard VPN will be inactive in this browser for all websites, even those from the website exclusion list.

:::

### AdGuard VPN is active for all apps except those from the the app exclusion list, and only for the websites from the website exclusion list

VPN mode:

* Active for all apps except app exclusions
* Active only for websites from website exclusions

Applied exclusion lists:

* App exclusion list [VPN active everywhere]
* Website exclusion list [VPN active selectively]

> To switch to this mode, leave AdGuard VPN active for all apps and set it to be on only for the websites from the website exclusion list. This mode provides even less VPN protection coverage because you need to add websites to the website exclusion list to ensure they are protected.

Reviewer: а еще нужно добавлять домены приложений, чтобы они шли через впн

This mode requires your attention to setting up your website exclusion list.

> **Empty exclusion lists:** If the **website exclusion list** is **empty**, AdGuard VPN will be **inactive for all websites**.

Reviewer: впн не работает нигде

---
title: VPN modes and Exclusion lists
sidebar_position: 11
---

## VPN modes and Exclusion lists

> :::note

Author: OK

This section explains how to use website and app exclusions in different VPN modes.

:::

## VPN modes

AdGuard VPN has two modes:

> 1. (Default) AdGuard VPN is active for all websites and apps, except for those added to the website and app exclusion lists. 

Author: По скобкам согласна. По второму изменению не согласна, так как хочу (пусть и избыточно) указывать названия списков

> 2. AdGuard VPN is active *selectively*: only for the websites and apps added to the website and app exclusion lists.

Author: Тут не согласна, так как хочу четко разделить листы

> 2. AdGuard VPN is active *selectively*: only for the websites and apps added to the website and app exclusion lists.

Reviewer 2: может тогда to the corresponding lists of exclusions?

> 2. AdGuard VPN is active *selectively*: only for the websites and apps added to the website and app exclusion lists.

Author: Давай я напишу тогда совсем просто:
`2. AdGuard VPN is active *selectively*: only for the websites and apps from the exclusion lists.`

:::note

> You can set *different* VPN modes for apps and websites. For example, you can set AdGuard VPN to be active for all apps except for those apps added to the app exclusion list, and active only for the websites from the website exclusion list.

Author: Уберу италик и заменю those apps на apps

:::

## Exclusion lists

> An exclusion list is a list of apps or websites that AdGuard VPN treats differently than others.

Author: OK

> Exclusion lists help you fine-tune the VPN activity for specific apps or websites.

Author: OK

:::note

Each VPN mode has its own app exclusion list and website exclusion list. 
In total, there are four exclusion lists:

> * App exclusion list [VPN active everywhere]

Reviewer 2: я бы скорее сказала, что у каждого режима два списка исключений — отдельный для сайтов и приложений
зачем усложнять?

Author: В том-то и беда, что у нас четыре списка, так как пока визуально
App exclusion list или Website exclusion list визуально не меняются для пользователя при переключении режима. И можно думать, что этот один список, и их всего два (один для приложений и один для сайтов)

И пока мы это интерфейсно не решим, придется так нудно писать

Reviewer 2: можно сказать, что в каждом режиме два списка исключений — для сайтов и приложений

Author: Хорошо, поправлю так

* App exclusion list [VPN active selectively]
* Website exclusion list [VPN active everywhere]
* Website exclusion list [VPN active selectively]

:::

## Combining VPN modes and Exclusion lists

> We will look at the different VPN activity setups in order of decreasing VPN coverage.

Reviewer 2: decreasing VPN coverage как-то сложно звучит

Author: OK, переделаю

### AdGuard VPN is active everywhere except for apps and websites from the app and website exclusion lists

VPN mode:
* Active for all apps except app exclusions
* Active for all websites except website exclusions

Applied exclusion lists:
* App exclusion list [VPN active everywhere]
* Website exclusion list [VPN active everywhere]

This is the default VPN mode with maximum VPN protection. If you want to exclude a website or an app from AdGuard VPN, add them to the exclusion list.

**Empty exclusion lists:** Your app exclusion list and website exclusion list can be empty. If nothing is specified in the exclusion lists, AdGuard VPN will be active for all apps and websites.

> **Why use this mode:** You want a blanket VPN protection for everything on your device. If a specific app or website needs to work outside the VPN, add it to exclusions.

Reviewer 2: to Exclusions или to the exclusions

Author: OK

:::caution

If you add a browser app to the app exclusion list, AdGuard VPN won't be active for all websites opened in that browser.

:::

> ### AdGuard VPN is active (1) only for the apps from the the app exclusion list, and (2) for all websites except those from the website exclusion list

Reviewer 2: цифры в скобках в заголовке кажутся не лучшим решением

Author: Уберу

VPN mode:
* Active only for apps from app exclusions
* Active for all websites except website exclusions

Applied exclusion lists:
* App exclusion list [VPN active selectively]
* Website exclusion list [VPN active everywhere]

> To get into this mode, you leave AdGuard VPN active for all websites and set it to be active only for the apps from the app exclusion list. This mode provides less VPN protection coverage as you need to add apps to the app exclusion list to ensure they are protected.

Reviewer 2: ```suggestion
To switch to this mode, leave AdGuard VPN enabled for all websites and set it to enable only for the apps from the app exclusion list. This mode provides less VPN protection coverage because you need to add apps to the app exclusion list to ensure they are protected.
```

Author: Поправлю всё, кроме enabled (заменю второе active на on)

This mode requires your attention to setting up your app exclusion list. 

**Empty exclusion lists:** If the **app exclusion list** is **empty**, AdGuard VPN will be **inactive for both websites and apps**.

**Why use this mode:** You need a VPN for most websites, but only for a few apps.

:::caution

If you don’t add a browser app to the app exclusion list, AdGuard VPN will be inactive in this browser for all websites, even those from the website exclusion list.

:::

### AdGuard VPN is active (1) for all apps except those from the the app exclusion list, and (2) only for the websites from the website exclusion list

VPN mode:
* Active for all apps except app exclusions
* Active only for websites from website exclusions

Applied exclusion lists:
* App exclusion list [VPN active everywhere]
* Website exclusion list [VPN active selectively]

> To get into this mode, you leave AdGuard VPN active for all apps and set it to be active only for the websites from the website exclusion list. This mode provides even less VPN protection coverage as you need to add websites to the website exclusion list to ensure they are protected.

Reviewer 2: аналогичная правка той, что выше

Author: OK

This mode requires your attention to setting up your website exclusion list. 

**Empty exclusion lists:** If the **website exclusion list** is **empty**, AdGuard VPN will be **inactive for all websites**.

**Why use this mode:** You need a VPN for most apps, but only for a few websites.

> 

Reviewer 2: лишние пустые строки

Author: OK

---
title: VPN modes and Exclusion lists
sidebar_position: 11
---

This section explains how to use website and app exclusions in different VPN modes.

## VPN modes

AdGuard VPN has two modes:

1. By default, AdGuard VPN is active for all websites and apps, except for those added to the website and app exclusion lists.

2. AdGuard VPN is active *selectively*: only for the websites and apps from the exclusion lists.

:::note

You can set different VPN modes for apps and websites. For example, you can set AdGuard VPN to be active for all apps except for those added to the app exclusion list, and active only for the websites from the website exclusion list.

:::

## Exclusion lists

> An exclusion list is a list of apps or websites that AdGuard VPN excludes or includes in its tunnel based on the current VPN mode.

Author: Нет, не обязательно в таком контексте

:::note

Each VPN mode has two exclusion lists: one for apps and one for websites.

> In total, there are four exclusion lists:

Author: OK

- App exclusion list [VPN active everywhere]
- App exclusion list [VPN active selectively]
- Website exclusion list [VPN active everywhere]
- Website exclusion list [VPN active selectively]

:::

## Combining VPN modes and exclusion lists

You can set up four combinations of VPN activity.

### Setup 1

AdGuard VPN is active everywhere except for apps and websites from the app and website exclusion lists.

VPN mode:

- Active for all apps except app exclusions
- Active for all websites except website exclusions

Applied exclusion lists:

- App exclusion list [VPN active everywhere]
- Website exclusion list [VPN active everywhere]

> This is the default VPN mode with maximum VPN protection. If you want to exclude a website or an app from AdGuard VPN, add them to the exclusion list.

Author: OK

> **Empty exclusion lists:** Your app exclusion list and website exclusion list can be empty. If nothing is specified in the exclusion lists, AdGuard VPN will be active for all apps and websites.

Author: OK

**Why use this mode:** You want a blanket VPN protection for everything on your device. If a specific app or website needs to work outside the VPN, add it to Exclusions.

:::caution

> If you add a browser app to the app exclusion list, AdGuard VPN won't be active for all websites opened in that browser.

Author: omg, sorry
OK

:::

### Setup 2

> AdGuard VPN is active only for the apps from the the app exclusion list, and for all websites except those from the website exclusion list.

Author: Я ее поставила специально, чтобы выделить кусок про веб-сайт. Давай уберём, если не сработало

This mode provides even less VPN protection coverage because you need to add websites to the website exclusion list to ensure they are protected.

This mode requires your attention to setting up your website exclusion list.

**Empty exclusion lists:** If the **website exclusion list** is **empty**, AdGuard VPN will be **inactive for all websites and apps**.

**Why use this mode:** You need a VPN for most apps, but only for a few websites.

:::caution

For all apps: make sure to add their respective domains to the website exclusion list. For example, for Facebook, add `facebook.com` to the website exclusion list.

:::

### Setup 4

AdGuard VPN is active only for the apps and websites from the app and website exclusion lists.

VPN mode:

- Active only for apps from app exclusions
- Active only for websites from website exclusions

Applied exclusion lists:

- App exclusion list [VPN active selectively]
- Website exclusion list [VPN active selectively]

You need to switch to this mode both for apps and websites. This mode provides the least VPN protection coverage.

> This mode requires your maximum attention to setting up your app and website exclusion lists.

Author: OK

:::

## Combining VPN modes and Exclusion lists

> The four VPN activity setups provide different levels of VPN coverage, from maximum to limited.

Reviewer 2: всё ещё звучит сложно
